hooks/wallet/WalletAPI.ts

```diff
@@ -78,6 +78,8 @@ export function useWalletAPI (): typeof WalletAPI {
     async function loadWallet (): Promise<void> {
       if (await hasMnemonicWallet()) {
         jellyfishWallet = await getMnemonicWallet(client, network as NetworkName)
+        const address = await jellyfishWallet.get(0).getAddress()
+        dispatch(wallet.actions.setAddress(address))
```

Comment:
not a good idea, we should not cache addresses as the wallet is agnostically provided. we will keep this for now